packages/frontend/components/Form/Select.tsx

```diff
@@ -94,9 +88,20 @@ export const PlanSelect: React.FC<PlanSelectProps> = ({
   if (isNumeric) {
     selectedValue = value ? value.toString() : null;
   }
-  const selectedOption =
-    selectOptions.find((option: any) => option.value === selectedValue) ??
-    noValueOption;
+  const selectedOption = selectOptions.find(
+    (option: any) => option.value === selectedValue
+  );
+
+  const styles = {
+    control: (baseStyles: any) => ({
+      ...baseStyles,
+      borderColor: "#e7ebf1",
+    }),
+    dropdownIndicator: (baseStyles: any, state: { isFocused: any }) => ({
+      ...baseStyles,
+      color: state.isFocused ? "#7586a0" : "#b4bbc8",
```

Comment:
Our custom styles dont seem to work with react-select's styling so I had to hard code these colors